python/mrc/core/segment.cpp

```diff
@@ -134,6 +134,12 @@ PYBIND11_MODULE(segment, py_mod)
                                                                         const std::string&,
                                                                         py::function)>(&BuilderProxy::make_source));
 
+    Builder.def("make_subscriber_source",
```

Comment:
Probably would be better to call it `make_source_with_subscriber` or just `make_source_subscriber` so that `make_source` and `make_source_XXX` are near eachother in intellisense.